src/main/java/com/coherentsolutions/training/aqa/java/zhavrid/util/PropertiesFileReader.java

```diff
@@ -0,0 +1,32 @@
+package com.coherentsolutions.training.aqa.java.zhavrid.util;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertiesFileReader {
+
+    private static Properties prop;
+    private static final String PROPERTIES_PATH = "src/main/resources/testdata.properties";
+
+    private PropertiesFileReader() {
+    }
+
+    public static synchronized Properties getProperties() {
+        if (prop == null) {
+            try (InputStream input = new FileInputStream(PROPERTIES_PATH)) {
+                prop = new Properties();
+                prop.load(input);
+            } catch (IOException ex) {
+                ex.printStackTrace();
```

Comment:
better to throw new RuntimeException with the message like "Cannot read property file %s", path
unless you'll have NPE while attempt to get a property from broken file